frontend/public/components/graphs/prometheus-poll-hook.ts

```diff
@@ -3,10 +3,11 @@ import { useCallback, useState } from 'react';
 
 import { usePoll, useSafeFetch } from '../utils';
 import { getPrometheusURL, PrometheusEndpoint } from './helpers';
+import { PrometheusResponse } from '.';
 
-const DEFAULT_DELAY = 15000; // 15 seconds
-const DEFAULT_SAMPLES = 60;
-const DEFAULT_TIMESPAN = 60 * 60 * 1000; // 1 hour
+const DEFAULT_DELAY: number = 15000; // 15 seconds
```

Comment:
I suggest not specifying types for simple constants like these since TypeScript can infer the type.